new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/InstallParams.java

```diff
@@ -49,6 +49,7 @@ public InstallParams version(String version) {
   }
 
   public String version() {
+    version = Installer.getActualVersionIfNeeded(location, type, version);
```

Comment:
This looks like a simple fluent getter but is doing a lot more.  Let's make the method more descriptive of what is going on.

Reply:
Okay